Rakefile

```diff
@@ -72,12 +72,7 @@ end
 
 desc 'Run brakeman ... use brakewan -I to add new ignores'
 task :brakeman do
-  begin
-    File.symlink('plugins', 'engines')
-    sh "brakeman --exit-on-warn --table-width 500"
-  ensure
-    File.unlink('engines')
-  end
+  sh "brakeman --exit-on-warn --table-width 500 --add-engine-path 'plugins/*'"
```

Comment:
:+1: You should try the plain text output with `-f plain` then you don't have to worry about `--table-width`